src/config.c

```diff
@@ -672,6 +672,7 @@ int config_write_template(const char *file, const struct config *cfg)
 	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "cons" MOD_EXT "\n");
 	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "evdev" MOD_EXT "\n");
 	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "httpd" MOD_EXT "\n");
+	(void)re_fprintf(f, "#module\t\t\t" MOD_PRE "mqtt" MOD_EXT "\n");
```

Comment:
could you please move this to the application section instead (module_app) ?

from the mqtt module:

```
const struct mod_export DECL_EXPORTS(mqtt) = {
       "mqtt",
       "application",
       module_init,
       module_close
};
```

Reply:
Aye aye, captain! Sorry about the unintentional misplacement!